modules/openxr/scene/openxr_hand.h

```diff
@@ -55,20 +55,32 @@ class OpenXRHand : public Node3D {
 		MOTION_RANGE_MAX
 	};
 
+	enum SkeletonRig {
+		SKELETON_RIG_OPENXR,
+		SKELETON_RIG_HUMANOID,
+		SKELETON_RIG_MAX
+	};
+
 private:
+	struct JointData {
```

Comment:
I believe parentage data is already in the Godot Humanoid data structure. Wondering if this is necessary duplication.

Reply:
This joint-data describes the `xr-joint` to `skeleton-bone-index` to `parent-xr-joint` relationship built dynamically for the current skeleton including handle of missing intermediate bones and when a bone is a child of a larger avatar. I don't believe the static Godot Humanoid data would be applicable here.